tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -120,17 +150,34 @@ def test_multiple_efs(
         remote_command_executor.run_remote_command(f"cat {existing_efs_mount_dirs[i]}/{existing_efs_filenames[i]}")
 
     all_mount_dirs = existing_efs_mount_dirs + new_efs_mount_dirs
-    _check_efs_correctly_mounted_and_shared(all_mount_dirs, remote_command_executor, scheduler_commands)
+    # append false for the one new_efs_mount_dir
+    iam_authorizations.append(False)
+    encryption_in_transits.append(False)
+    _check_efs_correctly_mounted_and_shared(
+        all_mount_dirs, remote_command_executor, scheduler_commands, iam_authorizations, encryption_in_transits
+    )
 
     if scheduler == "slurm":  # Only Slurm supports compute nodes reboot
         remote_command_executor, scheduler_commands = _check_efs_after_nodes_reboot(
-            all_mount_dirs, cluster, remote_command_executor, scheduler_commands_factory
+            all_mount_dirs,
+            cluster,
+            remote_command_executor,
+            scheduler_commands_factory,
+            iam_authorizations,
+            encryption_in_transits,
         )
 
     run_benchmarks(remote_command_executor, scheduler_commands)
 
 
-def _check_efs_after_nodes_reboot(all_mount_dirs, cluster, remote_command_executor, scheduler_commands_factory):
+def _check_efs_after_nodes_reboot(
+    all_mount_dirs,
+    cluster,
+    remote_command_executor,
+    scheduler_commands_factory,
+    iam_authorizations=None,
+    encryption_in_transits=None,
```

Comment:
Can we remove the None defaults?

Reply:
No, see [similar comment](https://github.com/aws/aws-parallelcluster/pull/4668/files/694ca2322f14403838d9cd068fd26a28c36b900e#r1044716143)

Reply:
Yes, see [action and explanation](https://github.com/aws/aws-parallelcluster/pull/4668#discussion_r1044786785)